frontend/packages/lint-config/package.json

```diff
@@ -30,15 +30,15 @@
     "@eslint/js": "^9.15.0",
     "@tsconfig/node22": "^22.0.4",
     "eslint": "^9.15.0",
-    "eslint-import-resolver-typescript": "^3.6.3",
-    "eslint-plugin-import-x": "^4.4.3",
+    "eslint-import-resolver-typescript": "^4.4.0",
+    "eslint-plugin-import-x": "^4.16.1",
```

Comment:
These were upgraded because the Typescript upgrade necessitated these to be upgraded too, but you don't see these errors unless you ran the linter.